docs/import-from-whitehall.md

```diff
@@ -0,0 +1,110 @@
+# Importing documents from Whitehall to Content Publisher
+
+There are two ways to import documents from Whitehall:
+- a single document
+- documents of a specific type or subtypes from a single publishing organisation
+
+## Running an import
+
+### Single document import
+
+This will import a single document and all its editions from Whitehall into Content Publisher.
+
+```
+rake import:whitehall_document[1234]
+```
+
+Where `1234` is the Whitehall document ID.  After running the rake task, a URL will be printed to the console which allows the status of import to be examined.
+
+Note that the ID in Whitehall's internal URLs is the edition number (e.g. for `/government/admin/publications/1034451` the edition is `1034451`).  The document ID can be obtained on a Rails console, using:
+
+```
+Edition.find(1034451).document.id
+```
+
+### Import documents of a specific type (and optional subtypes) from a single publishing organisation
+
+To import a single document type (e.g. news articles) from a single publisher (e.g. NDA):
+
+```
+rake import:whitehall_migration["nuclear-decommissioning-authority","news_article"]
+```
+
+To import multiple document subtypes (e.g. news stories and press releases) from a single publisher (e.g. NDA):
+
+```
+rake import:whitehall_migration["nuclear-decommissioning-authority","news_article","news_story,press_release"]
+```
+
+There is no limit to the number of subtypes (additional values are added comma separated).
+
+It is also possible to import a single document subtype (e.g. press releases) from a single publisher (e.g. NDA):
+
+```
+rake import:whitehall_migration["nuclear-decommissioning-authority","news_article","press_release"]
+```
+
+After running the rake task, a URL will be printed to the console which allows the status of import to be examined.
+
+## Running a migration locally using govuk-docker
+
+Using [govuk-docker](https://github.com/alphagov/govuk-docker), you can run an import locally without needing to replicate data.  In order to do this, you will need to override the [Plek](https://github.com/alphagov/plek) URL for Whitehall and Publishing API to point to staging.  Additionally, you will require the bearer tokens for these applications from [govuk-secrets](https://github.com/alphagov/govuk-secrets).
```

Comment:
Is there a particular reason we suggest staging rather than integration?

Reply:
I was previously having trouble accessing Publishing API in integration.

However, it appears I should've been using https://publishing-api.blue.integration.govuk.digital rather than https://publishing-api.integration.publishing.service.gov.uk.  Have updated the document to now use integration.